base/src/main/java/org/aya/tyck/LittleTyper.java

```diff
@@ -80,6 +84,18 @@ public record LittleTyper(@NotNull TyckState state, @NotNull LocalCtx localCtx)
       case FormTerm.PartTy ty -> FormTerm.Univ.ZERO;
       case IntroTerm.HappyPartEl el -> new FormTerm.PartTy(el.rhsType(), el.restr());
       case IntroTerm.SadPartEl el -> new FormTerm.PartTy(term(el), el.restr());
+      case FormTerm.Path path -> FormTerm.Univ.ZERO;
+      case IntroTerm.PathLam lam -> new FormTerm.Path(new Cube<>(
+        lam.params().map(Term.Param::ref),
+        term(lam.body()),
+        ImmutableSeq.empty() // TODO: clauses???
```

Comment:
I think `FormTerm.Path` should essentially contain a `PartEl` not a list of clauses. This means we can have trivial `Path`s, which is desirable over here.

Reply:
Now the path type `Cube` holds a generalized partial element `Partial` which can be either happy or sad.